.idea/codeStyles/Project.xml

```diff
@@ -1,8 +1,14 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <AndroidXmlCodeStyleSettings>
+      <option name="ARRANGEMENT_SETTINGS_MIGRATED_TO_191" value="true" />
+    </AndroidXmlCodeStyleSettings>
     <JetCodeStyleSettings>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
+    <MarkdownNavigatorCodeStyleSettings>
+      <option name="RIGHT_MARGIN" value="72" />
+    </MarkdownNavigatorCodeStyleSettings>
```

Comment:
I hope this is not a problem, but let me know if this does not work for you guys.

Reply:
@pfieffer Why the right margin here is 72? I think we can use nodes and connect them in the constraint layout.

Reply:
This is not actually related to `ConstraintLayout` or `Views` in anyway. As you can see this is in the `.idea/codeStyles/Project.xml ` file, which is a style file for the code. This is specifically for when the IDE encounters markdown files. This pieced of code style applies a right margin value of 72 characters for Markdown files.

Reply:
Gotcha! Thanks for the clarification.